scadalts-ui/.gitignore

```diff
@@ -29,5 +29,7 @@ test-results.xml
 *.sw?
 
 node_modules/*
-!@min-gb/
-
+!@min-gb/dist
+!@min-gb/public
+!@min-gb/src
+!@min-gb/*.*
```

Comment:
I copied min-gb and called min-gb2 and this pattern worked as we expected: (what has already been added to the git repository does not work with gitginore)
```
node_modules/*
!node_modules/@min-gb/
node_modules/@min-gb/vuejs-components/*
!node_modules/@min-gb/vuejs-components/dist
!node_modules/@min-gb/vuejs-components/src
!node_modules/@min-gb/vuejs-components/public
!node_modules/@min-gb/vuejs-components/*.*
```

Reply:
Sorry, I didn't notice there was a vuejs-components subfolder

Reply:
Hi @YuaFox, 
You can also add this to the main gitignore file (\Scada-LTS\.gitignore):
WebContent/resources/js-ui/

Reply:
It's already there
![image](https://github.com/user-attachments/assets/3c6e287d-4df0-4c2e-a90b-103d603a8382)

Reply:
@YuaFox,
Okay, but I guess this pattern doesn't work, see for yourself... maybe because at the beginning there is /

Reply:
Its working for me doing this command to force retracking of all files. 

```
git rm --cached -r . && git add .
```

After doing this i have 0 files tracked from WebContent/resources/js-ui/ folder, but i found a problem:
.gitignore says /tomcat/lib should not be tracked but its in the repo

Reply:
Hi @YuaFox Ok, it actually works for me, it doesn't show these changes either. (/WebContent/resources/js-ui/)
However, tracking /tomcat/lib is enabled, if it is in the sources, there is not much other option... and I don't see a pattern that could exclude this folder from tracking, there is neither the word lib nor tomcat.